aws/table_aws_ssm_document_permission.go

```diff
@@ -0,0 +1,133 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	"github.com/aws/aws-sdk-go-v2/service/ssm/types"
+
+	ssmv1 "github.com/aws/aws-sdk-go/service/ssm"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableAwsSSMDocumentPermission(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ssm_document_permission",
+		Description: "AWS SSM Document Permission",
+		List: &plugin.ListConfig{
+			Hydrate: listAwsSSMDocumentPermissions,
```

Comment:
Should this have a required key qual, or could we use parent hydrate? How many docs are common in a region?

Reply:
@cbruno10 My first thought was to use parent hydrate for this table, but it took over 25 minutes to get the result. There are around 15000 parent resources. Therefore I substituted needed quals for parent hydrate.